tests/chocolatey-tests/commands/choco-upgrade.Tests.ps1

```diff
@@ -406,44 +403,31 @@ To upgrade a local, or remote file, you may use:
-        # removed. Consider the test partially broken.
-        It "Keeps file '<_>' in the lib directory" -ForEach @('hasinnoinstaller.nuspec', 'hasinnoinstaller.nupkg', 'tools\chocolateyinstall.ps1', 'tools\chocolateyuninstall.ps1', 'tools\helloworld-1.0.0.exe', 'tools\helloworld-1.0.0.exe.ignore') -Tag FossOnly {
+        It "Keeps file '<_>' in the lib directory" -ForEach @('hasinnoinstaller.nuspec', 'hasinnoinstaller.nupkg', 'tools\chocolateyinstall.ps1', 'tools\chocolateyuninstall.ps1', 'tools\helloworld-1.0.0.exe', 'tools\helloworld-1.0.0.exe.ignore') {
             "$env:ChocolateyInstall\lib\$PackageUnderTest\$_" | Should -Exist
         }
 
         It "Doesn't keep a package backup in lib-bkp" {
             "$env:ChocolateyInstall\lib-bkp\$PackageUnderTest" | Should -Not -Exist
         }
 
-        # Only two files are backed up as we was not able to download and extract the new package
-        # This does not work as expected when runnning in test kitchen and with Chocolatey Licensed Extension installed.
-        # It is believed to be a problem with test kitchen, or the VM that we are using that is causing the issue.
-        # The result is that versioned backup files of each file in the package is created, instead of the package being
-        # removed. Consider the test partially broken.
-        It "Creates backup of file '<_>' in lib-bad" -ForEach @('.chocolateyPending', 'tools\a-locked-file.txt') -Tag FossOnly {
+        It "Creates backup of file '<_>' in lib-bad" -ForEach @('.chocolateyPending', 'tools\a-locked-file.txt') {
             "$env:ChocolateyInstall\lib-bad\$PackageUnderTest\$PackageVersion\$_" | Should -Exist
         }
 
-        It "Did not create backup of file '<_>' in lib-bad" -ForEach @('hasinnoinstaller.nuspec', 'hasinnoinstaller.nupkg', 'tools\chocolateyinstall.ps1', 'tools\chocolateyuninstall.ps1', 'tools\helloworld-1.0.0.exe', 'helloworld-1.0.0.exe.ignore') {
+        It "Did not create backup of file '<_>' in lib-bad" -ForEach @('hasinnoinstaller.nuspec', 'tools\chocolateyinstall.ps1', 'tools\chocolateyuninstall.ps1', 'tools\helloworld-1.0.0.exe', 'helloworld-1.0.0.exe.ignore') {
             "$env:ChocolateyInstall\lib-bad\$PackageUnderTest\$PackageVersion\$_" | Should -Not -Exist
         }
 
-        # This does not work as expected when runnning in test kitchen and with Chocolatey Licensed Extension installed.
-        # It is believed to be a problem with test kitchen, or the VM that we are using that is causing the issue.
-        # The result is that versioned backup files of each file in the package is created, instead of the package being
-        # removed. Consider the test partially broken.
-        It "Outputs a message showing that installation failed." -Tag FossOnly {
+        It "Did not create backup of file 'hasinnoinstaller.nupkg' in lib-bad" -Tag FossOnly {
+            "$env:ChocolateyInstall\lib-bad\$PackageUnderTest\$PackageVersion\$_" | Should -Not Exist
+        }

```

Comment:
```suggestion
        It "Did not create backup of file 'hasinnoinstaller.nupkg' in lib-bad" -Tag FossOnly {
            "$env:ChocolateyInstall\lib-bad\$PackageUnderTest\$PackageVersion\hasinnoinstaller.nupkg" | Should -Not Exist
        }
```
There is no `$_` in this context because there's no `-ForEach`...

But: assuming the other tests changed to `--no-reduce`, would this one go away and the nupkg put back in the immediately preceding test?